### DISCUSSION TITLE
Got an error after the first chat

### DISCUSSION THREAD

**Discussion by fnakashima** [Zero6992/chatGPT-discord-bot#133] [Q&A] [answered]
Hi
I got the following error at the second chat while the first response was returned as expected.
-------------------------
This model's maximum context length is 4097 tokens, however you requested 4120 tokens (197 in your prompt; 3923 for the completion). Please reduce your prompt; or completion length.
-------------------------
I only entered a very short question. What is causing the issue and how can I solve it?

Thanks in advance.

**Comment by Zero6992** [comment, marked as answer]
https://github.com/acheong08/ChatGPT/issues/501

Same issue as here, I updated to the latest version

Please download again and then run `pip install -r requirements.txt`

**Reply by fnakashima** [reply]
Thanks for the quick fix!
Just confirmed the issue was fixed.

It looks like the way the response has changed. What else has changed?
